.github/workflows/checks.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: npm run test:generation
       - name: Compare gas costs
         uses: ./.github/actions/gas-compare
-        if: github.base_ref == 'master'
+        if: (github.event_name == 'pull_request' && github.base_ref == 'master') || (github.event_name == 'push' && github.ref_name == 'master')
```

Comment:
Do we want the gas comparison to only work for master ?

If we ever do a `v6-next` branch (or similar) I guess we will want the gas comparaison to run.

I'd just remove that line, and have the action always run.
```suggestion
```

Reply:
Note: this change (either removing the if or making it support push to master) should also be added to the "compare storage layout" action on line 74

Reply:
For the record, we discussed in a meeting that removing the check will allow comparisons to other branches, but it's likely that comparisons after major changes may fail (e.g. comparing `v6-next` vs. `master`). Not a problem for now.